docs/charts/doughnut.md

```diff
@@ -63,6 +63,7 @@ The doughnut/pie chart allows a number of properties to be specified for each da
 | [`hoverBackgroundColor`](#interations) | [`Color`](../general/colors.md) | Yes | Yes | `undefined`
 | [`hoverBorderColor`](#interactions) | [`Color`](../general/colors.md) | Yes | Yes | `undefined`
 | [`hoverBorderWidth`](#interactions) | `Number` | Yes | Yes | `undefined`
+| [`weight`](#styling) | `Number` | - | - | `1`
```

Comment:
In #6020, all basic types were converted to lower case. So, `Number` should be `number`.

Reply:
Thanks for the heads up.